erpnext/public/js/pos/pos.js

```diff
@@ -280,6 +280,7 @@ erpnext.pos.PointOfSale = Class.extend({
 	},
 	set_totals: function() {
 		var me = this;
+		this.wrapper.find(".base-total").text(format_currency(me.frm.doc["base_total"], me.frm.doc.currency));
```

Comment:
Please use `me.frm.doc["total"]`, instead of `me.frm.doc["base_total"]`.

`base_total` is the total in base currency.